components/gitpod-protocol/src/teams-projects-protocol.ts

```diff
@@ -152,3 +153,15 @@ export interface TeamMembershipInvite {
     /** This is a flag that triggers the HARD DELETION of this entity */
     deleted?: boolean;
 }
+
+export interface PrebuildEvent {
+    id: string;
+    creationTime: string;
+    status: WebhookEvent.Status | WebhookEvent.PrebuildStatus;
```

Comment:
Why not use the same interface we use for the DB? I feel that would avoid a lot of confusion, as that is our usual pattern (for the JSON RPC API). Plus it would avoid the duplication here. :lotus_position:

Reply:
I want to avoid pushing event payload over websocket too often. In some projects this may be a significant amount of data. Also, if's feels unnecessary to push user id's. 
In general, the pattern you are referring to caused some pain in the past where we lost control over what got serialized and transferred over the websocket.

Reply:
Ah, you're right, some fields we might definitely not want in here. :bulb: (e.g., authorizedUser)
Could we still make this a subshape with `Omit`/`Pick` then? really don't see the point in merging both status fields here. :upside_down_face: And it nicely expresses the relationship.

Reply:
Merging status here, as from prebuild UI perspective there is little need to deal with two different statuses at all. Actually we're mostly interested in prebuild status and for error cases in the generic ones. What's the point having it separated for the frontend?

Reply:
That we can choose where to show it. E.g. I'm pretty sure we want a column status, and maybe a small area underneath for details if it's an error state (401, for instance). But I'm absolutely not sure we want to show an message related to prebuilds in that same area, maybe even not on the same page. :cloud: